src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -13,7 +13,7 @@ import { debug } from '../debug-operator';
 @Component({
   template: `
     <div *ngIf="displayDashboard">
-      <span *ngIf="planetType !== 'community'">
+      <span *ngIf="planetType === 'community'">
```

Comment:
I don't think we want this change (we should only see the request button from nations & centers)